protocols/1607/cell_culture_assay.ot2.py

```diff
@@ -0,0 +1,119 @@
+from opentrons import labware, instruments, modules, robot
+from otcustomizers import StringSelection
+from opentrons.legacy_api.modules import tempdeck
+import math
+
+"""Controlling multiple of the same module in the protocol:
+1. SSH into robot
+2. run $ls /dev/tty*
+   you are looking for two values with the format /dev/ttyACM*
+   you will use those values in lines 24-27.
+If you need to know which tempdeck is hooked up to which port:
+1. unplug one of the modules
+2. run $ls /dev/tty* : the results correlates to the module that is plugged in
+3. plug the other module in and run ls /dev/tty* again, you will be able to
+   know the value of the second module
+"""
+
+# defining two Temperature Modules
+tempmod1 = tempdeck.TempDeck()
+tempmod2 = tempdeck.TempDeck()
+tempmod3 = tempdeck.TempDeck()
+tempmod4 = tempdeck.TempDeck()
+
+tempmod1._port = '/dev/ttyACM1'
+tempmod2._port = '/dev/ttyACM2'
+tempmod3._port = '/dev/ttyACM3'
+tempmod4._port = '/dev/ttyACM4'
+
+if not robot.is_simulating():
+    tempmod1.connect()
+    tempmod2.connect()
+    tempmod3.connect()
+    tempmod4.connect()
+
+metadata = {
+    'protocolName': 'Cell Culture Assay',
+    'author': 'Nick <protocols@opentrons.com>',
+    'source': 'Custom Protocol Request'
+}
+
+# create custom plate
+plate_name = 'Cellvis-24-flat'
+if plate_name not in labware.list():
+    labware.create(
+        plate_name,
+        grid=(6, 4),
+        spacing=(19.3, 19.3),
+        diameter=16.2,
+        depth=19,
+        volume=3600
+    )
+
+# load labware and modules
+tempmod1 = modules.load('tempdeck', '10')
```

Comment:
The name here cannot be the same as the one in line 19, they are two different things.
You can call this `temp_mod_1`

Reply:
To call the methods of the modules, you need to use the name in line 19 rather than this one